### DISCUSSION TITLE
what is the 'CC upgrade' means?

### DISCUSSION THREAD

**Discussion by KIMQI001** [filecoin-project/lotus#5927] [Storage Provider] [answered]
does the 'CC upgrade' make any difference?
I wander does it means this sector will be a sector that can accept deals?
I make a deal, and CC upgrade a sector ,but lotus-miner new a sector to make the deal .

**Comment by magik6k** [comment, marked as answer]
CC upgrade means reusing collateral from a non-deal (CC) sector for a sector with deals. When you do CC upgrade, the old sector will expire on the next proving deadline, and the new sector with deals will get power.

You can see that sector was CC-upgrading another one by looking at the precommit message - the `ReplaceSector` field will be set to the sector which is being upgraded.
